packages/amplify-codegen-model-plugin/src/plugin.ts

```diff
@@ -0,0 +1,51 @@
+import { PluginFunction, Types } from '@graphql-codegen/plugin-helpers';
+import { GraphQLSchema, parse, visit } from 'graphql';
+import { printSchemaWithDirectives } from 'graphql-toolkit';
+import { AppSyncSwiftVisitor } from './visitors/appsync-swift-visitor';
+import { RawAppSyncLocalConfig } from './visitors/appsync-visitor';
+import { AppSyncJSONVisitor } from './visitors/appsync-json-metadata-visitor';
+import { AppSyncJavaVisitor } from './visitors/appsync-java-visitor';
+import { AppSyncLocalTypeScriptVisitor } from './visitors/appsync-typescript-visitor';
```

Comment:
Are we keeping "AppSyncLocal" naming around?

Reply:
good catch